index.js

```diff
@@ -805,159 +818,205 @@ function buildValue (location, input) {
     type = 'string'
   }
 
-  switch (type) {
-    case 'null':
-      code += 'json += serializer.asNull()'
-      break
-    case 'string': {
-      funcName = nullable ? 'serializer.asStringNullable.bind(serializer)' : 'serializer.asString.bind(serializer)'
-      code += `json += ${funcName}(${input})`
-      break
+  if ('const' in schema) {
```

Comment:
I would add an option to turn this feature on, or it will be a major version with a harder distribution across userland